### DISCUSSION TITLE
Able to download External Subtitles

### DISCUSSION THREAD

**Discussion by David-Moreira** [jellyfin/jellyfin-meta#78] [Proposals]
This is a continuation of the proposal on https://github.com/jellyfin/jellyfin-web/issues/5618

> "Hello,
> I'd like to work on this feature
> https://features.jellyfin.org/posts/1181/subtitles-download
> 
> I believe just adding a download button by each external subtitle should be enough"
> ![image](https://github.com/jellyfin/jellyfin-meta/assets/22283161/e1f320e9-1c18-48ee-af41-473206ac3bea)

@thornbill :
> "Remote subtitles would be dealing with downloading subtitles from providers. Not downloading existing local subtitles. It would be best to open a [meta discussion](https://github.com/jellyfin/jellyfin-meta/discussions) for this."

@David-Moreira :
> "I'm sorry I do not understand your point. From the moment they are visible in the available subtitles screen I posted they are local and have been downloaded by jellyfin from a provider is that not the case? It's only a matter of downloading the local external. Subtitle."

I'd like to reiterate that from my point of view:
- The subtitles from the providers are already locally available, and it should be just a matter of triggering a download as opposed to having to do more complex handling, but maybe I am missing something, please let me know.

**Comment by amdnsr** [comment]
Certainly agree with you, and share the same thought. We can already download the files to our local devices, and not having the subtitles in our local device is a big issue especially if the movie is in a foreign language, and there is no option to download the subtitles when you are actually watching the movie, like in a long flight. The whole point of downloading the mp4 file becomes useless if the subtitles are not there.

There's a feature request for this too.
https://features.jellyfin.org/posts/1316/download-locally-available-subtitle
